### PR TITLE
Cat.Count Response

### PR DESCRIPTION
**Pull request by philkra**
as titled.

### CODE REVIEW

**Comment by delvedor** [comment]
The error will be fixed once https://github.com/elastic/clients-flight-recorder/pull/645 is merged.

**Comment by philkra** [comment]
@delvedor , I confirm the issue is fixed and this PR is ready for review! Thanks for the quick fix

File: specification/specs/cat/cat_count/CatCountRecord.ts [hunk 1]
```diff
@@ -19,6 +19,6 @@
 
 class CatCountRecord {
   count: string
-  epoch: string
-  timestamp: string
+  epoch?: string
```
Comment (delvedor) [inline specification/specs/cat/cat_count/CatCountRecord.ts:22]:
```suggestion
  epoch?: EpochMillis
```

File: specification/specs/cat/cat_count/CatCountRecord.ts [hunk 2]
```diff
@@ -19,6 +19,6 @@
...
+  timestamp?: string
```
Comment (delvedor) [inline specification/specs/cat/cat_count/CatCountRecord.ts:23]:
```suggestion
  timestamp?: DateString
```

**Comment by delvedor** [comment]
https://github.com/elastic/elastic-client-generator/pull/117#pullrequestreview-593917682 still needs to be addressed. Can you regenerate the output as well?

**Comment by philkra** [comment]
@delvedor , I generated a new output - I needed to merge master though, because the `EpochMillis` type was not know yet in this branch

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**